components/log-viewer-webui/server/src/fastify-v2/plugins/app/socket/MongoSocketIoServer/index.ts

```
@@ -301,26 +311,26 @@ class MongoSocketIoServer {
         }
 
-        this.#fastify.log.debug(
+        this.#logger.debug(
             "Query ID to query hash map:" +
             ` ${JSON.stringify(Array.from(this.#queryIdToQueryHashMap.entries()))}`
         );
 
         if (false === collection.isReferenced()) {
-            this.#fastify.log.debug(`Collection:${queryParams.collectionName}` +
+            this.#logger.debug(`Collection:${queryParams.collectionName}` +
             " deallocated from server.");
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)